From d5e8f58887c572e3d9317b68999ca5d6320f2815 Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Wed, 15 Jul 2015 14:52:04 +0200
Subject: [PATCH] Revert "journald-audit: exit gracefully in the case we can't
 join audit multicast group"

This reverts commit 9b5e05005e534fc7fb6dc56c94e3296bb17fe122.

Cherry-picked from:
Resolves: #1222517
---
 src/journal/journald-audit.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/src/journal/journald-audit.c b/src/journal/journald-audit.c
index 77abe2e63..46eb82fa3 100644
--- a/src/journal/journald-audit.c
+++ b/src/journal/journald-audit.c
@@ -533,10 +533,8 @@ int server_open_audit(Server *s) {
                 }
 
                 r = bind(s->audit_fd, &sa.sa, sizeof(sa.nl));
-                if (r < 0) {
-                        log_warning_errno(errno, "Failed to join audit multicast group, ignoring: %m");
-                        return 0;
-                }
+                if (r < 0)
+                        return log_error_errno(errno, "Failed to join audit multicast group: %m");
         } else
                 fd_nonblock(s->audit_fd, 1);
 
